### PR TITLE
Handle `.wtf` command without query

### PR DESCRIPTION
**Pull request by brad90four**
-Added a default value to `query`, and if the value is None, send the
link to the repository in an embed.
-Modified the if / else statements to if / elif / else with the new
if statement to check if the value of `query` is None.

## Relevant Issues

Closes#938

## Description

-Added a default value to `query`, and if the value is None, send the
link to the repository in an embed.
-Modified the if / else statements to if / elif / else with the new
if statement to check if the value of `query` is None.
## Did you:

- [X] Join the [**Python Discord Community**](https://discord.gg/python)?
- [X] Read all the comments in this template?
- [X] Ensure there is an issue open, or link relevant discord discussions?
- [X] Read and agree to the [contributing guidelines](https://pythondiscord.com/pages/contributing/contributing-guidelines/)?

### CODE REVIEW

File: bot/exts/utilities/wtf_python.py [hunk 1]
```diff
@@ -79,15 +79,26 @@ def fuzzy_match_header(self, query: str) -> Optional[str]:
...
+                title="WTF Python?!",
+                colour=constants.Colours.dark_green,
+                description="A repository filled with suprising snippets that can make you say WTF?! "
+                f"[Go to the Repository]({BASE_URL})"
+            )
+            logo = File(LOGO_PATH, filename="wtf_logo.jpg")
+            no_query_embed.set_thumbnail(url="attachment://wtf_logo.jpg")
+            await ctx.send(embed=no_query_embed, file=logo)
+            return
+        elif len(query) > 50:
```
[the remark this answers is not in the document]
Comment (brad90four) [reply bot/exts/utilities/wtf_python.py]: Yeah I had that originally and then got carried away a bit. Should be fixed now.

**Review by ChrisLovering — APPROVED** [review]

File: bot/exts/utilities/wtf_python.py [hunk 2]
```diff
@@ -79,14 +79,26 @@ def fuzzy_match_header(self, query: str) -> Optional[str]:
         return match if certainty > MINIMUM_CERTAINTY else None
 
     @commands.command(aliases=("wtf", "WTF"))
-    async def wtf_python(self, ctx: commands.Context, *, query: str) -> None:
+    async def wtf_python(self, ctx: commands.Context, *, query: str = None) -> None:
```
Comment (hedyhli) [inline bot/exts/utilities/wtf_python.py:82]: Pretty trivial but let's update the type hint here
```suggestion
    async def wtf_python(self, ctx: commands.Context, *, query: Optional[str] = None) -> None:
```
Apart from that, LGTM 👍

**Review by hedyhli — CHANGES_REQUESTED** [review]

File: bot/exts/utilities/wtf_python.py (hunk 2 above)
Comment (brad90four) [reply bot/exts/utilities/wtf_python.py]: Fixed with [31152da](https://github.com/python-discord/sir-lancebot/pull/939/commits/31152dae2702203aa0ba3b37baaa3a8fea3ff86f)

File: bot/exts/utilities/wtf_python.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (brad90four) [reply bot/exts/utilities/wtf_python.py]: Fixed with [6f16669](https://github.com/python-discord/sir-lancebot/pull/939/commits/6f1666990ed8c3e590fba8a0193d836d87723f14)

**Review by Qwerty-133 — COMMENTED** [review]
I think the output looks just a tad nicer when the "go to repository" link is on a line by it's own:
image

**Comment by brad90four** [comment]
> 
> 
> I think the output looks just a tad nicer when the "go to repository" link is on a line by it's own: image

I agree! Added a single new line character, but haven't tested locally.

File: bot/exts/utilities/wtf_python.py [hunk 3]
```diff
@@ -79,14 +79,26 @@ def fuzzy_match_header(self, query: str) -> Optional[str]:
...
 
         Gets the link of the fuzzy matched query from https://github.com/satwikkansal/wtfpython.
         Usage:
             --> .wtf wild imports
         """
+        if query is None:
+            no_query_embed = Embed(
+                title="WTF Python?!",
+                colour=constants.Colours.dark_green,
+                description="A repository filled with suprising snippets that can make you say WTF?!\n"
```
Comment (hedyhli) [inline bot/exts/utilities/wtf_python.py:94]: Currently it looks like this:
image
```suggestion
                description="A repository filled with suprising snippets that can make you say WTF?!\n\n"
```
I think double newline looks a bit better. Perhaps we could switch to the discord link button (like `.challenge`) when we start using it more often in other places, but this is good for now.

**Review by hedyhli — CHANGES_REQUESTED** [review]

**Review by hedyhli — APPROVED** [review]
Thanks!

**Review by Qwerty-133 — APPROVED** [review]
Everything's wrapped up, thanks for implementing this!

**Pull request merged**